src/contributor.js

```diff
@@ -267,7 +267,7 @@ let contributors = [
     id: 27,
     name: "Ronnie Maynard",
     cohort: "2023 Alpha",
-    bio: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
+    bio: "I am a full stack web developer with a background as an Army Veteran and a blue-collar worker. Tech has always been my passion, and being a student at LEARN and an intern for LEARN, helping develop this application, has allowed me to explore that passion and fully immerse myself in the world of tech.",
```

Comment:
"background as an Army Veteran" sounds a little off to me. Maybe either "developer and Army Veteran" or "background as an Army paratrooper" (forgive me if that is not your correct job title)